vyper/utils.py

```diff
@@ -226,15 +229,11 @@ def is_varname_valid(varname, custom_units, custom_structs, constants):
         return False, f"Duplicate name: {varname}, previously defined as a struct."
     if varname in constants:
         return False, f"Duplicate name: {varname}, previously defined as a constant."
-    if varname_lower in BASE_TYPES:
-        return False, f"{varname} name is a base type."
-    if varname_lower in VALID_UNITS:
-        return False, f"{varname} is a built in unit type."
-    if varname_lower in RESERVED_KEYWORDS:
-        return False, f"{varname} is a a reserved keyword."
-    if varname_upper in OPCODES:
+    if varname_lower in map(lambda k: k.lower(), RESERVED_KEYWORDS):
```

Comment:
I think list comprehensions are a bit cleaner here
```suggestion
    if varname_lower in [k.lower() for k in RESERVED_KEYWORDS]:
```